src/components/StandardFacetContent.tsx

```diff
@@ -0,0 +1,39 @@
+import { FilterGroup } from './FilterGroup';
+import { DisplayableFacet } from '@yext/search-headless-react';
+import { StandardFacetProps } from './FacetProps';
+
+/**
+ * A component that displays the content of a standard facet.
+ *
+ * @param props - props to render the component
+ * @returns A React component for the content of a standard facet
+ *
+ * @internal
+ */
+export function StandardFacetContent({
```

Comment:
is this exported as StandardFacet anywhere?  I see it in the Stories byt don't see it elsewhere (i'm probably missing something)

Reply:
I'm exporting a StandardFacet inside Facets, which basically just passes the props through and is different from this one. The StandardFacetContent needs to take in a facet so I made it internal and separate from the StandardFacet. Let me know if that's fine with you!